scripts/pre_commit_linter.py

```diff
@@ -88,8 +88,8 @@
     'utf', 'pylint:', 'http://', 'https://', 'scripts/', 'extract_node']
 
 EXCLUDED_PATHS = (
-    'third_party/*', 'build/*', '.git/*', '*.pyc', 'CHANGELOG',
-    'integrations/*', 'integrations_dev/*', '*.svg', '*.gif',
+    'third_party/*', 'build/*', '.codeclimate.yml', '.git/*', '*.pyc',
```

Comment:
Why do you need to add .codeclimate.yml here but not (e.g.) .travis.yml?

Reply:
.codeclimate.yml contains definition for bad pattern checks and at the time of linting they are being caught that's why I have to add this file here. While .travis.yml does not contain anything like that.